channeldb/graph.go

```diff
@@ -1023,6 +1023,10 @@ func (c *ChannelGraph) AddChannelEdge(edge *models.ChannelEdgeInfo,
 	}
 
 	for _, f := range op {
+		if f == nil {
```

Comment:
I think if we wanna be safu we should return an error to the callsite

Reply:
Made it an error